### PR TITLE
website: Fix an erratic line brake in items list

### PR DESCRIPTION
**Pull request by jlduran**

### CODE REVIEW

**Comment by sergio-carlavilla** [comment]
Uo, good catch!

File: website/content/en/where.adoc [hunk 1]
```diff
@@ -156,9 +156,7 @@ Uppercase has special meaning. For example:
 * a first _beta release_ is not a (production) `RELEASE`.
 The word `CURRENT` is sometimes a source of confusion:
```
Comment (jlduran) [inline website/content/en/where.adoc:157]: I wonder if this sentence should be spaced before, so it is not inside the bullet? Looks a little strange.

Comment (sergio-carlavilla) [reply website/content/en/where.adoc:157]: Yes, seems we need an empty line there

**Comment by pauamma** [comment]
Ready for commit?

**Comment by dbaio** [comment]
Hello.

Thanks for your contribution.

To properly register your contribution, we will need your e-mail and name. This is what we are receiving [at this moment](https://patch-diff.githubusercontent.com/raw/freebsd/freebsd-doc/pull/296.patch):

```
Jose Luis Duran <jlduran@users.noreply.github.com>
```

Or, if you don't want to share your e-mail, no worries. We can add a line like this in the commit message:

```
Submitted by:   Jose Luis Duran
```

Regards.

**Comment by jlduran** [comment]
Sorry about that... It should be OK now.
Thank you!

**Comment by dbaio** [comment]
Committed on 771a52cc6181f40a4323189e0749d76f309e5ed8. Thanks for your contribution.

**Pull request closed without merging**